jwst/outlier_detection/tests/test_outlier_detection.py

```diff
@@ -261,7 +262,8 @@ def test_outlier_step_base(we_three_sci, tmp_cwd):
     assert len(median_files) != 0
 
 
-def test_outlier_step_spec(tmp_cwd, tmp_path):
+@pytest.mark.parametrize('resample', [True, False])
+def test_outlier_step_spec(tmp_cwd, tmp_path, resample):
```

Comment:
Since this test is being updated, can it be parametrized with 'save_intermediate_results=True/False`? I think that would remove some of the duplication and the need for `make_input` but I will leave that judgement up to you. I'm ok with how it looks in this PR but wanted to see what you think.

Reply:
Sure, let me take a look at that.

Reply:
Done - and thanks for the suggestion, that did clean it up a bit.